docs/SUMMARY.md

```diff
@@ -7,15 +7,15 @@
 * [What is Concrete?](get-started/readme.md)
 * [Installation](get-started/installing.md)
 * [Quick start](get-started/quick_start.md)
-* [Compatibility](get-started/compatibility.md)
+* [Limitations](get-started/limitations.md)
 * [Terminology](get-started/terminology.md)
 
-## Core features
+## Operations
 
 * [Overview](core-features/fhe_basics.md)
```

Comment:
I'm not a fan of this overview here as the first document in "Operations". It's too long and not directly related to the contents of 'Operation'
I would suggest keep the first part "Operations on encrypted values" part, and add more explanations of different types of operations inside Concrete that are discussed in "Operation part",  such as:

> In Concrete, there are basically two types of operations:
> linear operations, like additions, subtraction and multiplication by an integer, which are very fast
> and all the rest, which is done by a table lookup (TLU).
> TLU are essential to be able to compile all functions, by keeping the semantic of user's program, but they can be slower, depending on the bitwidth of the inputs of the TLU.

Then the rest parts in "Overview":  "Noise and Bootstrap", "Probability of Error", "Function evaluation", "PBS management", "Development Workflow", they can be either simplified into "Get Started", or moved to "Explanations".

Let me know what you think @bcm-at-zama? ( It can be for the next PR)

Reply:
Yes, but it's not part of this PR. It's related to https://github.com/zama-ai/concrete-internal/issues/812 (part 4). So, if you don't mind, I would resolve this comment and keep it for future in 812